locale/de/manageiq.po

```diff
@@ -10601,9 +10604,15 @@ msgstr "ConfigurationScript-Basis|Ausgabe"
 msgid "ConfigurationScriptBase|Payload"
 msgstr "ConfigurationScript-Basis|Nutzdaten"
 
+msgid "ConfigurationScriptBase|Payload error"
+msgstr "ConfigurationScript-Basis | Fehler bei Nutzdaten"
```

Comment:
I think the spaces here might be incorrect.  The `|` is not part of the string, but is a parsing separator.

Reply:
Also the model name should not be translated.  So, in total this should be

```suggestion
msgstr "ConfigurationScriptBase|Fehler bei Nutzdaten"
```

Reply:
It should be trivial to add specs that check this, so I'm wondering if we should do that.

Reply:
FWIW in gettext terminology this prefix thing is called a "context".  See here for an example of it in the gettext ruby gem README: https://github.com/ruby-gettext/gettext#sp_-or-spgettext-context-aware-pluralized

Another link: https://www.gnu.org/software/gettext/manual/html_node/Contexts.html